src/app/Fake.DotNet.Cli/CreateProcessExt.fs

```diff
@@ -0,0 +1,121 @@
+namespace Fake.Core
+
+open Fake.DotNet
+
+/// Some extensions for the `CreateProcess` module, opened automatically (use add `open Fake.Core`)
+[<AutoOpen>]
+module CreateProcessDotNetExt =
```

Comment:
Note: Not a huge fan of AutoOpen (see guidelines). Will try to suggest something else, just wanted to let you know early.

Reply:
Just following the precedent of [the definition of `withFramework`](url) here, for sake of consistency.

Reply:
Yes but first it is part of the `Fake.Core.Process` module and second it doesn't contain "new" types and other stuff, maybe we can at least move that outside of the "AutoOpen". Let me take a look

Reply:
Moving to an explicit module `Fake,DotNet.Create Process` would be little hardship, if that's more in line with the current style.

Reply:
Yes that is my current thinking it if could just look like:

```fsharp

    Command.RawCommand("localtool", Arguments.OfArgs ["arg1"; "arg2"])
    |> CreateProcess.fromCommand
    // I feel the name is a bit too long but to get the idea
    |> DotNetCreateProcess.with buildOptions // prepare to execute dotnet localtool.
    |> Proc.run
    |> ignore
```

Also from a design perspective, I'm not sure why we need to extend the `Proc` module because you can always just edit the `CreateProcess` instance?

Like I said I will play around with the code now (These are my current thinkings, please feel free to add yours or (dis)agree).

Reply:
Another thing I'm thinking about is that `withDotNetTool` is basically just prefixing everything with "dotnet.exe", I'm not sure if that is what you usually want because usually I feel like you want to replace the first argument as well when using the function (ie when the tool has a different name to the exe file)?